### DISCUSSION TITLE
You don't have access to Github Copilot

### DISCUSSION THREAD

**Discussion by zyxdSTU** [community/community#118665] [Copilot Conversations]
### Select Topic Area

Bug

### Body

Yesterday, I purchased a copilot subscription, which includes a one month usage qualification.
### "Your copilot" page as follow picture
![image](https://github.com/community/community/assets/26239665/4e24e8a9-42ec-4a98-bded-037f840244bf)

### "Billing and plans" as follow picture
![image](https://github.com/community/community/assets/26239665/ebcb64ee-9516-4b4d-a9ea-cb9b14969595)

### I found this issue when logging into the copilot plugin in Idea， 
![image](https://github.com/community/community/assets/26239665/4074c286-7f2f-4153-b442-f3bab567cf20)
### and the content of idea.log is 
``2024-04-11 15:24:29,375 [ 692981]   INFO - #copilot - [auth] Invalid copilot token: missing token: 403 ``
### Also, I would like to add that I am in China and not on the trade control list
I reinstalled the plugin many times
I feel a lot of pain not being able to use Copilot,
help me!! help me!!!

**Comment by grbzemir** [comment]
Hello [zyxdSTU](https://github.com/zyxdSTU), I'm get in some unknown reason when i try to auth the device, means i can get the device code and open the github site,but when i paste the device code and click auth confirm button, idea still show me need to login to github, even i used the http proxy, could u help me out. really appreciate that.

best regards
